src/components/chat/UserList.js

```diff
@@ -59,8 +57,10 @@ class UserList extends Component {
           {viewing.map((username, i) => {
             return (
               <li key={i}>
-                <ViewingUserIcon />
-                {username}
+                <ViewingUserIcon
+                  username={username}
+                  isCurrentUser={username === currentUsername}
```

Comment:
@jimmcgaw I know what's happening -- ^^this^^ sort of line needs to be added to the 2 other statuses! `viewing` is the green dot, so when their dot is yellow, `isCurrentUser` doesn't get set and so ` (me)` doesn't show up.

Reply:
Fixed in https://github.com/cryptag/leapchat/pull/277